### DISCUSSION TITLE
Clone GSA Shop Packs from one game to another

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#752] [Suggestions]
Based on the discussion in the discord. For those of us who have to create a whole lot of ASA shop packs, the ability to simply clone one ASE shop pack to “convert” it for ASA would be a tremendous blessing. Let the users worry about making sure commands and images are correct; I just want to avoid the labor of meticulously setting up so many of the same packs again.

**Comment by babsgosgens** [comment]
This seems related to #893

**Comment by birdman3131** [comment]
So this is now doable. Export as CSV. Change the base game ID and remove the ID for the Shop-Pack and it will upload it as new packs for the other game.
